app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource
+
+    def index
+      @conferences_available = Conference.with_roles([:admin, :organizer, :cfp], current_user)
```

Comment:
I think it would make more sense for the index action of the CommentsController to set a @comments variable with the comments that we want available (those comments are Comment.accessible_by(current_ability), and I think it should work fine).